vortexasdk/enums.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum
+
+
+class VesselAttributesEnum(str, Enum):
+    FlagTag = "vessel_flag_tag"
+    ScrubberTag = "vessel_scrubber_tag"
+
+
+class ScrubbersFittedEnum(str, Enum):
+    Off = ("off",)
+    Included = ("inc",)
+    Excluded = "exc"
```

Comment:
I love these enums, but I sadly I don' think we can use them.

Enums mean that the SDK is dependent on the values returned by the  Vortexa API, which sits outside of any semver schema changes.

For example, the API could return a new ScrubbersFitted option, `foo`, without any major, minor, or even patch semver change. This would raise an exception in the SDK when the SDK attempts to serialize `foo` into the `ScrubbersFittedEnum`. For this reason, we're safer off sticking to strings in cases like these 👍

Reply:
^^ The above statement is actually incorrect. Python can serialize a string into an non-existing enum.